src/elements/kw-projects-header/kw-projects-header.html

```diff
@@ -112,7 +113,7 @@
                 height: 44px;
                 border-radius: 50%;
                 border: 2px solid #fee33e;
-                z-index: 0;
+                z-index: -1;
```

Comment:
the z-index -1 is for the :after, not the link (a pulsating circle around the icon). it's an unfortunate kind of :after as it's on top of the element, but I think this was the best solution